1-exercises/B-object-arrays/exercise-2.js

```diff
@@ -16,10 +16,18 @@ let store = [
 
 let money = 1.5;
 
-let thingsICanBuy =
+let thingsICanBuy = store.map(item => {
+  for(let item of store) {
```

Comment:
Deago, I trust that this works but it looks unorthodox.
There really is no need to use a 'for of' loop.
I believe we are meant to solve this task using map and filter.
Have a look at the solution by CYF